### PR TITLE
Fix github page workflow

### PR DESCRIPTION
**Pull request by mcbarton**
# Description

Please include a summary of changes, motivation and context for this PR.

@anutosh491  The Github page workflow was broken when we stopped using condas emsdk. This PR should fix that.

Fixes # (issue)

## Type of change

Please tick all options which are relevant.

- [x] Bug fix
- [ ] New feature
- [ ] Added/removed dependencies
- [ ] Required documentation updates

### CODE REVIEW

File: .github/workflows/deploy-github-page.yml
Comment (anutosh491) [inline .github/workflows/deploy-github-page.yml:50]: Can this be removed ? Feels like duplication to me.

**Review by anutosh491 — APPROVED** [review]
Mostly looks good except my comment. 
Let's wait for the ci to be green before merging !

File: .github/workflows/deploy-github-page.yml
Comment (mcbarton) [reply .github/workflows/deploy-github-page.yml:50]: This is not duplicating anything as far as I can see. This is a separate workflow and setups emsdk the same way main does.

Comment (anutosh491) [reply .github/workflows/deploy-github-page.yml:50]: Wait I am confused. Looking into your file I see this 
```
      - name: Setup emsdk
        shell: bash -l {0}
        run: |
          cd $HOME
          git clone https://github.com/emscripten-core/emsdk.git
          cd emsdk
          ./emsdk install  ${{ matrix.emsdk_ver }}

      - name: Setup emsdk
        shell: bash -l {0}
        run: |
          emsdk install ${{matrix.emsdk_ver}}   
          
```
I am just guessing we don't need to do this twice ?

Comment (anutosh491) [reply .github/workflows/deploy-github-page.yml:50]: cc @mcbarton Am I missing something ? Check my comment above

**Pull request merged**
